<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-0.9.305">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Vincent van Hoef">

<title>Tumor progression in breast cancer reflected in gene expression patterns and mutational profile</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link id="quarto-text-highlighting-styles" href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet">
<script src="index_files/libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="index_files/libs/jquery-3.5.1/jquery.min.js"></script>
<link href="index_files/libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="index_files/libs/datatables-binding-0.18/datatables.js"></script>
<link href="index_files/libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="index_files/libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="index_files/libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<script src="index_files/libs/jszip-1.10.20/jszip.min.js"></script>
<link href="index_files/libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet">
<script src="index_files/libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="index_files/libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="index_files/libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="index_files/libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="index_files/libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="index_files/libs/dt-ext-scroller-1.10.20/css/scroller.dataTables.min.css" rel="stylesheet">
<script src="index_files/libs/dt-ext-scroller-1.10.20/js/dataTables.scroller.min.js"></script>
<link href="index_files/libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet">
<script src="index_files/libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="./assets/style.css">
</head>

<body>

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Tumor progression in breast cancer reflected in gene expression patterns and mutational profile</h1>
            <p class="subtitle lead">Project #5975</p>
                </div>
  </div>
    
  



  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Vincent van Hoef </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">5/9/22</p>
      </div>
    </div>
      
    </div>
    
  
  </header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#material-and-methods" id="toc-material-and-methods" class="nav-link" data-scroll-target="#material-and-methods">Material and Methods</a>
  <ul class="collapse">
  <li><a href="#preprocessing" id="toc-preprocessing" class="nav-link" data-scroll-target="#preprocessing">Preprocessing</a></li>
  <li><a href="#variant-calling" id="toc-variant-calling" class="nav-link" data-scroll-target="#variant-calling">Variant Calling</a></li>
  <li><a href="#filtering" id="toc-filtering" class="nav-link" data-scroll-target="#filtering">Filtering</a></li>
  <li><a href="#copy-number-analysis" id="toc-copy-number-analysis" class="nav-link" data-scroll-target="#copy-number-analysis">Copy Number Analysis</a></li>
  <li><a href="#tumor-mutational-burden" id="toc-tumor-mutational-burden" class="nav-link" data-scroll-target="#tumor-mutational-burden">Tumor Mutational Burden</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#quality-control" id="toc-quality-control" class="nav-link" data-scroll-target="#quality-control">Quality control</a>
  <ul class="collapse">
  <li><a href="#fastqc" id="toc-fastqc" class="nav-link" data-scroll-target="#fastqc">FastQC</a></li>
  <li><a href="#coverage" id="toc-coverage" class="nav-link" data-scroll-target="#coverage">Coverage</a></li>
  </ul></li>
  <li><a href="#variant-calling-1" id="toc-variant-calling-1" class="nav-link" data-scroll-target="#variant-calling-1">Variant Calling</a></li>
  <li><a href="#tumor-mutational-burden-1" id="toc-tumor-mutational-burden-1" class="nav-link" data-scroll-target="#tumor-mutational-burden-1">Tumor Mutational Burden</a></li>
  <li><a href="#copy-number-alteration" id="toc-copy-number-alteration" class="nav-link" data-scroll-target="#copy-number-alteration">Copy Number Alteration</a></li>
  <li><a href="#nanostring-data" id="toc-nanostring-data" class="nav-link" data-scroll-target="#nanostring-data">Nanostring data</a></li>
  <li><a href="#survival-data" id="toc-survival-data" class="nav-link" data-scroll-target="#survival-data">Survival Data</a></li>
  </ul></li>
  <li><a href="#data-responsibility" id="toc-data-responsibility" class="nav-link" data-scroll-target="#data-responsibility">Data Responsibility</a></li>
  <li><a href="#acknowledgements" id="toc-acknowledgements" class="nav-link" data-scroll-target="#acknowledgements">Acknowledgements</a></li>
  <li><a href="#closing-procedures" id="toc-closing-procedures" class="nav-link" data-scroll-target="#closing-procedures">Closing Procedures</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
</div>
<main class="content quarto-banner-title-block page-columns page-full" id="quarto-document-content">
<!-- Add icon library -->
<link rel="stylesheet" href="./assets/font-awesome-4.7.0/css/font-awesome.min.css">




<div class="cell">

</div>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>This is the report of the analysis for project #5975, requested by Lisa Rydén (LU). For this study, targeted next generation sequencing by hybridization capture was performed to get data about a panel of approx 450 genes. The main aim of the analysis is to call somatic variants in a collection of 205 breast cancer samples; some of which are matched pairs of primary tumors and distance metastases or primary tumors and lymph node metastases. Matched normal samples are however not available but might be included later for more accurate results.</p>
<p>Contract:</p>
<ul>
<li>Preprocessing of 205 samples</li>
<li>CNV and SNV detection</li>
<li>Format data to allow for analysis of correlation with survival and clinical annotation</li>
<li>Detection of differences in CNV and SNV between primary tumors and lymph node or distant metastases</li>
<li>Correlation with tumor grouping and BC360 gene signatures based on Nanostring expression data</li>
</ul>
<p>[150 hours]</p>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>If the visualization in this report do not work as expected, you might have to update or change your browser. Please let me know if any plots seem strange or absent.</p>
</div>
</div>
<p>205 samples are included in this study. [below table contains 207 entries, but 2 samples were not sequenced.]</p>
<div class="cell">
<div class="cell-output-display">
<div id="htmlwidget-45314fc6a80c2808caa6" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-45314fc6a80c2808caa6">{"x":{"filter":"none","extensions":["Buttons","Scroller"],"data":[[1,2,7,8,9,10,11,12,13,15,17,19,20,21,22,23,24,25,26,27,28,29,31,32,33,35,36,38,41,42,43,45,46,47,49,50,51,52,53,54,55,56,57,60,61,62,64,65,66,67,68,69,70,71,73,74,75,77,79,81,83,84,85,86,88,89,90,92,96,97,98,100,101,104,105,108,109,110,112,114,115,116,117,119,121,122,123,125,130,131,134,135,138,139,140,144,145,147,148,149,150,151,152,154,155,156,157,159,160,161,162,163,165,166,167,168,169,170,171,172,174,176,177,178,184,185,186,189,191,192,193,194,195,198,200,203,204,205,206,207,208,209,210,211,212,213,214,215,217,218,221,224,228,229,230,232,235,236,238,240,248,250,251,252,253,254,255,259,261,262,263,264,269,273,274,277,278,279,281,282,283,284,285,286,289,291,293,294,295,297,302,306,307,311,312,314,316,317,318,319,327,328,329,330,331,332,333],["L011","L012","L070","M224","L054","L050","L060","L060","L062","L044","L044","L078","L024","L030","L001","L001","L022","L022","L004","L079","L079","L003","L034","L028","L028","L025","L033","L040","L070","L064","L083","L040","L067","L034","M248","M248","M219","M210","M210","M237","M237","M203","M203","M209","M214","M214","M201","M216","M216","L038","L038","M229","M229","M252","M253","L003","L030","L002","L063","L012","L040","L067","M253","M250","M206","M206","M205","M221","M221","M235","M235","M241","M226","M205","M201","M263","M224","M242","M264","M212","M212","L012","L070","L025","L062","L004","L086","L011","L057","L057","M246","M242","L057","L062","L073","L079","L083","M264","L018","L078","L024","L065","L064","L056","L002","L009","L020","L028","L060","L026","L027","M244","L072","M263","M202","M202","M250","L072","L072","L016","L068","L068","L043","L054","M222","M226","L018","L086","L082","L082","L078","L065","L008","L077","L050","L043","M222","H411","H411","H401","H401","H404","H404","H413","H413","H413","H408","H408","H403","H403","H405","M202","M213","L053","M235","M243","L077","L082","L087","L077","L052","L036","M229","L059","H405","H406","H406","M222","M252","L056","L002","L004","H408","H407","M209","M213","M216","M219","M232","M236","M254","M256","M261","M240","M225","M241","M254","M254","M257","M256","M253","L010","L030","L036","L065","L010","L016","L024","M242","M255","L008","L008","L043","M246","M225","L053","M243"],["PT","PT","PT","PT","PT","PT","PT","LNM","LNM","PT","LNM","PT","PT","LNM","PT","LNM","LNM","PT","LNM","LNM","PT","LNM","LNM","PT","LNM","REC","REC","REC","REC","PT","PT","PT","PT","PT","PT","LNM","PT","PT","LNM","PT","LNM","PT","LNM","LNM","PT","LNM","PT","PT","LNM","PT","LNM","PT","LNM","PT","LNM","PT","PT","LNM","REC","LNM","LNM","LNM","PT","LNM","PT","LNM","PT","PT","REC","PT","LNM","PT","REC","REC","REC","PT","LNM","PT","PT","LNM","PT","REC","LNM","PT","PT","PT","PT","REC","PT","LNM","PT","LNM","REC","REC","REC","REC","REC","REC","LNM","LNM","LNM","PT","REC","PT","REC","REC","REC","REC","REC","REC","REC","REC","REC","REC","PT","LNM","PT","PT","LNM","PT","PT","LNM","PT","LNM","LNM","PT","PT","LNM","PT","LNM","REC","LNM","REC","LNM","REC","REC","REC","PT","LNM","PT","LNM","PT","LNM","PT","LNM","REC","PT","LNM","PT","LNM","LNM","REC","REC","PT","REC","REC","REC","REC","REC","PT","REC","PT","REC","REC","PT","PT","LNM","PT","REC","LNM","PT","REC","REC","REC","PT","PT","REC","REC","REC","REC","REC","REC","REC","REC","REC","REC","PT","LNM","REC","PT","REC","PT","REC","REC","REC","REC","REC","REC","REC","REC","PT","LNM","LNM","LNM","PT","REC","PT"],["PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","DM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+DM","PT+DM+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM","PT+DM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+DM","PT+DM","PT+DM","PT+DM","PT+LNM","PT+DM+LNM","PT+DM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","DM","PT+DM+LNM","PT+DM","PT+DM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+LNM","PT+DM+LNM","DM","DM","PT+DM+LNM","PT+DM+LNM","DM","DM","DM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+LNM","PT+LNM","PT+DM+LNM","PT+LNM","PT+DM+LNM","PT+DM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM","PT+DM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM+LNM","DM","PT+DM+LNM","DM","PT+DM","PT+DM+LNM","DM","PT+LNM","PT+LNM","PT+LNM","PT+DM+LNM","PT+DM","PT+LNM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","DM","PT+LNM","PT+DM","PT+DM+LNM","PT+DM","DM","DM","PT+DM+LNM","PT+DM","DM","DM","PT+DM","PT+DM","PT+DM+LNM","PT+DM+LNM","DM","PT+DM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM","PT+DM+LNM","PT+DM","PT+DM","PT+DM+LNM","PT+DM+LNM","DM","PT+DM+LNM","PT+DM+LNM","PT+DM+LNM","PT+LNM","PT+DM","PT+DM","PT+DM"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Sample ID<\/th>\n      <th>Studiekod<\/th>\n      <th>Tumor location<\/th>\n      <th>Matched or not<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel"],"deferRender":true,"scrollY":200,"scroller":true,"columnDefs":[{"className":"dt-right","targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="material-and-methods" class="level1 page-columns page-full">
<h1>Material and Methods</h1>
<section id="preprocessing" class="level2">
<h2 class="anchored" data-anchor-id="preprocessing">Preprocessing</h2>
<p>Raw reads were trimmed using TrimGalore v0.6.1 prior to mapping to GRCh37 using bwa mem of bwa v0.7.17. Subsequently, duplicates were tagged and removed using MarkDuplicates from picard v2.23.4. Base recalibration was performed and applied using GATK v4.2.0 with known sites datasets dbsnp_138, 1000G_phase1 and Mills_and_1000G_gold_Standard_indels.</p>
</section>
<section id="variant-calling" class="level2">
<h2 class="anchored" data-anchor-id="variant-calling">Variant Calling</h2>
<p>Somatic variant calling of all samples was performed by combining the results of three variant callers; i.e.&nbsp;Mutect2 v4.2.0, VarScan v2.4.2 and freebayes v1.3.2. These were chosen because they are capable of running without a matched normal sample. Mutect2 was supplied with a generic panel-of-normals (“somatic-b37-Mutect2-exome-panel”) and a germline resource (“gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only”). VarScan was run with standard parameters and freebayes was run with -F 0.01 -C 5 –pooled-continuous and subsequently filtered to retain calls with QUAL &gt; 20. Variant calling was restricted to the list of targets used for the hybrid capture. Variants found in all three callers were called again using the <em>call</em> command from bcftools 1.14 and annotated with VEP v99 using the <em>vcf2maf.pl</em> script from vcf2maf v1.6.21.</p>
<p>Additionally, somatic calling for the distant and lymph node metastases was done in a more rigorous manner. For these samples, the matched primary tumor was used as a <em>pseudo</em> normal sample and variant calling was performed by Mutect2 only. A custom panel-of-normals was created from the primary tumors using the <em>CreateSomaticPanelOfNormals</em> command from GATK. The same germline resource was used as the one employed for the primary tumor analysis. Contamination was estimated and Mutect2 calls were filtered as per the appropriate <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035531132">GATK workflow</a>. Keep in mind that this protocol only detects variants that are a variant in the metastatic sample but not in the primary tumor. It does not tell us anything about the variants of the primary tumor in itself; if both primary and metasasis have the variant it is not withheld in this analysis.</p>
</section>
<section id="filtering" class="level2">
<h2 class="anchored" data-anchor-id="filtering">Filtering</h2>
<p>In addition to taking the intersect of the results of the three variant callers, variants which are present at a variant allele frequency (VAF) &gt; 1% in the Genome Aggregation Database (gnomAD) were removed. Files with raw and filtered variant calls are provided witht his report.</p>
</section>
<section id="copy-number-analysis" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="copy-number-analysis">Copy Number Analysis</h2>
<p>For a rough estimation of the copy number alterations in the metastases, the R package pureCN was used <span class="citation" data-cites="riester_purecn:_2016">(<a href="#ref-riester_purecn:_2016" role="doc-biblioref">Riester et al.</a>)</span>. It was designed for hybrid capture sequencing data, especially with medium-sized targeted gene panels without matching normal samples in mind (matched whole-exome data is of course supported). The primary tumors were used to create a <em>pseudo</em> normal coverage profile which is then compared to the metastatic sample. Keep in mind that this means that any copy number alteration is a change in the metastasis as compared to the primary tumor. It does not tell us anything about the status of the primary tumor in intself.</p>
<div class="no-row-height column-margin column-container"><div id="ref-riester_purecn:_2016" class="csl-entry" role="doc-biblioentry">
Riester, Markus, et al. <span>“<span>PureCN</span>: Copy Number Calling and <span>SNV</span> Classification Using Targeted Short Read Sequencing.”</span> <em>Source Code for Biology and Medicine</em>, vol. 11, no. 1, Dec. 2016, p. 13, <a href="https://doi.org/10.1186/s13029-016-0060-z">https://doi.org/10.1186/s13029-016-0060-z</a>.
</div></div><p>Analysis of the copy number alterations in the primary tumors was considered to be too unreliable without a matched normal sample.</p>
</section>
<section id="tumor-mutational-burden" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="tumor-mutational-burden">Tumor Mutational Burden</h2>
<p>To calculate the TMB, the R package TMBLeR was used <span class="citation" data-cites="fancello_tmbler_2021">(<a href="#ref-fancello_tmbler_2021" role="doc-biblioref">Fancello et al.</a>)</span>. The TMB is defined here as the number of mutations per Mb, excluding known cancer mutations but including synonymous mutations. There are several ways to define and filter the TMB. The filtering used here is described as well in the TMBLeR’s manual.</p>
<div class="no-row-height column-margin column-container"><div id="ref-fancello_tmbler_2021" class="csl-entry" role="doc-biblioentry">
Fancello, Laura, et al. <span>“<span>TMBleR</span>, a Bioinformatic Tool to Optimize <span>TMB</span> Estimation and Predictive Power.”</span> <em>Bioinformatics (Oxford, England)</em>, Dec. 2021, p. btab836, <a href="https://doi.org/10.1093/bioinformatics/btab836">https://doi.org/10.1093/bioinformatics/btab836</a>.
</div></div></section>
</section>
<section id="results" class="level1 page-columns page-full">
<h1>Results</h1>
<p>Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Therefore, a quality control step was run on the raw reads before and after trimming by the tool <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastqQC</a>. Three samples are corrupted and were removed (11,12 and 19).</p>
<section id="quality-control" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="quality-control">Quality control</h2>
<section id="fastqc" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="fastqc">FastQC</h3>
<div class="panel-tabset column-body-outset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true">Before Trimming</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">After Trimming</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<p>FastQC was run on the raw sequences as they are delivered by the sequencing facility. Read1 and Read2 are considered separately. Here we mainly look for outlier samples.</p>
<div id="fig-pre-trim-qc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/QC/multiqc_report.html">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 1: Pre Trim QC</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<p>The tool was run again after removal of low quality bases and adapter sequences.</p>
<div id="fig-pre-trim-qc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/QC/multiqc_report_after_trim.html">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 2: Post Trim QC</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<p>Overall, the quality is sufficient to proceed with the analysis.</p>
</section>
<section id="coverage" class="level3">
<h3 class="anchored" data-anchor-id="coverage">Coverage</h3>
<p>The number of reads overlapping a specific nucleotide is an important quality metric because it will influence the confidence in the subsequent variant calls. Moreover, the more reads cover the target, the better we will be able to detect small clones. After mapping the trimmed reads to the genome, the average read depth per target (7455 targets in total) and per sample is calculated. The results per sample are shown in <a href="#fig-coverage-plot">Figure&nbsp;3</a>.</p>
<div id="fig-coverage-plot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="https://uppsala.app.box.com/embed/s/vidv6u2pfzxlr6rilt5w9evot29ywvcq?sortColumn=date&amp;view=list" style="width:100%; height:500px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 3: Coverage per Sample.</figcaption><p></p>
</figure>
</div>
<p>Sample 224 and 28 have rather low coverage (22x and 33x respectively), but are kept in for the further downstream analysis. But good to keep in mind. The other samples have an average coverage beween 250x and 500x which is sufficient for somatic variant calling in heterogeneous tumors.</p>
</section>
</section>
<section id="variant-calling-1" class="level2">
<h2 class="anchored" data-anchor-id="variant-calling-1">Variant Calling</h2>
<p>Here you can find the results of the variant calling analysis for the different sample groups.</p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-5-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-1" role="tab" aria-controls="tabset-5-1" aria-selected="true">Primary Tumor</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-2" role="tab" aria-controls="tabset-5-2" aria-selected="false">Distant Metastases</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-3" role="tab" aria-controls="tabset-5-3" aria-selected="false">Lymph Node Metastases</a></li></ul>
<div class="tab-content">
<div id="tabset-5-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-5-1-tab">
<p>The primary tumors have been called by three different variant callers and the intersect of the SNPs of these three tools was retained for further analysis. The standard format to store variant information is as VCF files.</p>
<p><a href="/Results/all_samples_freebayes.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Primary Tumor VCF Files - Freebayes only</button></a></p>
<p><a href="./Results/all_samples_varscan.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Primary Tumor VCF Files - VarScan only</button></a></p>
<p><a href="./Results/all_samples_mutect2.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Primary Tumor VCF Files - Mutect2 only</button></a></p>
<p><a href="./Results/all_samples/3_common_vcf.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Primary Tumor VCF Files - Combined Variants</button></a></p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">Overview</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">Oncoplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">Interaction</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div id="fig-pt-overview" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/Primary_Tumor_Summary_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 4: Overview of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div id="fig-pt-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 5: Oncoplot of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div id="fig-pt-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 6: Co-occurrence of common mutations.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</div>
<div id="tabset-5-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-2-tab">
<p><a href="./Results/distant_tumor_mutect2.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Distant Tumor VCF Files - Mutect2 only</button></a></p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">Overview</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">Oncoplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false">Interaction</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div id="fig-dt-overview" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/Primary_Tumor_Summary_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 7: Overview of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div id="fig-dt-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 8: Oncoplot of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div id="fig-dt-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 9: Co-occurrence of common mutations.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</div>
<div id="tabset-5-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-3-tab">
<p><a href="./Results/lymph_node_tumor_mutect2.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Lymph Node Tumor VCF Files - Mutect2 only</button></a></p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-4-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-1" role="tab" aria-controls="tabset-4-1" aria-selected="true">Overview</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-2" role="tab" aria-controls="tabset-4-2" aria-selected="false">Oncoplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-3" role="tab" aria-controls="tabset-4-3" aria-selected="false">Interaction</a></li></ul>
<div class="tab-content">
<div id="tabset-4-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-4-1-tab">
<div id="fig-lnm-overview" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/Primary_Tumor_Summary_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 10: Overview of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-4-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-2-tab">
<div id="fig-lnm-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 11: Oncoplot of Primary Tumor samples.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-4-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-3-tab">
<div id="fig-lnm-oncoplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf" style="width:100%; height:700px;">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 12: Co-occurrence of common mutations.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="tumor-mutational-burden-1" class="level2">
<h2 class="anchored" data-anchor-id="tumor-mutational-burden-1">Tumor Mutational Burden</h2>
<div class="cell">
<div class="cell-output-display">
<div id="htmlwidget-35402e0c0d9038ec6fe3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-35402e0c0d9038ec6fe3">{"x":{"filter":"none","extensions":["Buttons","Scroller"],"data":[[1,10,100,101,104,105,108,109,110,112,114,115,116,117,119,121,122,123,125,13,131,134,135,138,139,140,144,145,147,148,149,15,150,151,152,154,155,156,157,159,160,161,162,163,166,167,168,169,17,170,171,172,174,176,177,178,184,185,186,189,191,192,193,194,195,198,2,20,200,203,204,205,206,207,208,209,21,210,211,212,213,214,215,217,218,22,221,224,228,229,23,230,232,235,236,238,24,240,248,25,250,251,252,253,254,255,259,26,261,262,263,264,269,27,273,274,277,278,279,28,281,282,283,284,285,286,289,29,291,293,294,295,297,302,306,307,31,311,312,314,316,317,318,319,32,327,328,329,33,330,331,332,333,35,36,38,41,42,43,45,46,47,49,50,51,52,53,54,55,56,57,60,61,62,64,65,66,67,68,69,7,70,71,73,74,75,77,79,8,81,83,84,85,86,88,89,9,90,92,96,97,98],["vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts","vaf=0NoCancerMuts"],[1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452,1.286452],[780,575,530,510,489,551,537,548,555,518,541,539,523,529,528,518,519,534,534,632,537,509,553,533,510,546,501,540,523,512,503,574,485,476,496,526,559,546,557,515,507,513,527,533,534,551,540,536,586,509,527,536,539,550,553,527,538,518,505,527,508,509,490,512,548,487,812,583,553,561,530,518,532,533,529,499,554,497,534,531,526,472,487,532,537,560,512,1633,550,522,562,562,505,488,511,525,504,489,537,511,523,511,511,518,552,540,533,524,538,534,558,527,501,660,551,531,544,529,545,1199,520,577,547,530,557,511,489,519,539,548,548,489,533,538,553,561,553,541,463,544,572,496,579,494,521,567,589,570,510,511,493,539,555,519,582,507,483,529,562,514,576,516,563,563,561,537,564,548,549,535,525,538,525,526,515,522,511,533,584,508,577,513,543,531,510,530,556,550,624,652,517,590,528,525,538,530,607,494,539,543,543,556],[606.32,446.97,411.99,396.44,380.12,428.31,417.43,425.98,431.42,402.66,420.54,418.98,406.54,411.21,410.43,402.66,403.44,415.1,415.1,491.27,417.43,395.66,429.86,414.32,396.44,424.42,389.44,419.76,406.54,397.99,391,446.19,377.01,370.01,385.56,408.88,434.53,424.42,432.97,400.33,394.11,398.77,409.65,414.32,415.1,428.31,419.76,416.65,455.52,395.66,409.65,416.65,418.98,427.53,429.86,409.65,418.2,402.66,392.55,409.65,394.88,395.66,380.89,397.99,425.98,378.56,631.19,453.18,429.86,436.08,411.99,402.66,413.54,414.32,411.21,387.89,430.64,386.33,415.1,412.76,408.88,366.9,378.56,413.54,417.43,435.31,397.99,1269.38,427.53,405.77,436.86,436.86,392.55,379.34,397.22,408.1,391.78,380.12,417.43,397.22,406.54,397.22,397.22,402.66,429.09,419.76,414.32,407.32,418.2,415.1,433.75,409.65,389.44,513.04,428.31,412.76,422.87,411.21,423.65,932.02,404.21,448.52,425.2,411.99,432.97,397.22,380.12,403.44,418.98,425.98,425.98,380.12,414.32,418.2,429.86,436.08,429.86,420.54,359.9,422.87,444.63,385.56,450.08,384,404.99,440.75,457.85,443.08,396.44,397.22,383.22,418.98,431.42,403.44,452.41,394.11,375.45,411.21,436.86,399.55,447.74,401.1,437.64,437.64,436.08,417.43,438.42,425.98,426.76,415.87,408.1,418.2,408.1,408.88,400.33,405.77,397.22,414.32,453.96,394.88,448.52,398.77,422.09,412.76,396.44,411.99,432.2,427.53,485.06,506.82,401.88,458.63,410.43,408.1,418.2,411.99,471.84,384,418.98,422.09,422.09,432.2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Sample<\/th>\n      <th>Filter<\/th>\n      <th>Sequencing_Size<\/th>\n      <th>Tot_Number_Mutations<\/th>\n      <th>TMB_per_Mb<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel"],"deferRender":true,"scrollY":200,"scroller":true,"scrollX":true,"columnDefs":[{"className":"dt-right","targets":[0,2,3,4]}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="copy-number-alteration" class="level2">
<h2 class="anchored" data-anchor-id="copy-number-alteration">Copy Number Alteration</h2>
<p>Copy number alterations were estimated using the R package pureCN. In this analysis, metastases were compared to their matched primary tumor. It is important to keep in mind that therefore these results can be affected by the state of the primary tumor as well; e.g.&nbsp;an amplification in a lymph node tumor might indicate a deletion in the matched primary tumor rather than a genuine amplification.</p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-6-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-1" role="tab" aria-controls="tabset-6-1" aria-selected="true">Distant Metastases</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-2" role="tab" aria-controls="tabset-6-2" aria-selected="false">Lymph Node Metastases</a></li></ul>
<div class="tab-content">
<div id="tabset-6-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-6-1-tab">
<p>#<code>{r} #dmag &lt;- read.csv("./Results/CNA/distant_tumor/all_genes_alt.csv", sep = ";") #datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE)) #</code></p>
<p>The full results of the pureCN analysis per patient can be downloaded with the button below. There are several files per patient here and not all are even informative. The per gene statistics for each sample can for example be found in the *_genes.csv files. A description of the columns in this genes.csv file can be found in Table 2 of <a href="https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf">the pureCN manual</a>.</p>
<p><a href="https://uppsala.box.com/v/lymphnodecopynumberdata" download=""><button class="btn"><i class="fa fa-download"></i> Download Distant Metastases CNA data</button></a></p>
<div class="quarto-layout-panel" height="1500px">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-cna-dm-amp" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><embed src="./Results/CNA/distant_tumor/Deleted_in_distant_tumor_vs_primary_tumor.pdf" class="img-fluid" data-ref-parent="fig-cna-dm"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) Amplified</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-cna-dm-del" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><embed src="./Results/CNA/distant_tumor/Deleted_in_distant_tumor_vs_primary_tumor.pdf" class="img-fluid" data-ref-parent="fig-cna-dm"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) Deleted</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 13: Most Common Alterations in Distant Tumor versus Primary Tumor.</figcaption><p></p>
</figure>
</div>
<p>Here you can see the results per patient.</p>
<p>Here you can see the results per patient.</p>
<p><iframe src="https://uppsala.app.box.com/embed/s/h3rs5sctm0c559yw8qyer235deh22yri?sortColumn=date&amp;view=list" style="width:100%; height:700px;"></iframe></p>
</div>
<div id="tabset-6-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-2-tab">
<p>#<code>{r} #dmag &lt;- read.csv("./Results/CNA/distant_tumor/all_genes_alt.csv", sep = ";") #datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE)) #</code></p>
<p>The full results of the pureCN analysis per patient can be downloaded here. There are several files per patient here and not all are even informative. The per gene statistics for each sample can for example be found in the *_genes.csv files. A description of the columns in this genes.csv file can be found in Table 2 of <a href="https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf">the pureCN manual</a>.</p>
<p><a href="./Results/lymph_node_tumor_copy_number_data.tar.gz" download=""><button class="btn"><i class="fa fa-download"></i> Download Lymph Node Metastases CNA data</button></a></p>
<div class="quarto-layout-panel" height="1500px">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-cna-lnm-amp" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><embed src="./Results/CNA/lymph_node/Amplified_in_lymph_node_vs_primary_tumor.pdf" class="img-fluid" data-ref-parent="fig-cna-lnm"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) Amplified</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-cna-lnm-del" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><embed src="./Results/CNA/lymph_node/Deleted_in_lymph_node_vs_primary_tumor.pdf" class="img-fluid" data-ref-parent="fig-cna-lnm"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) Deleted</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 14: Most Common Alterations in Lymph Node Tumor versus Primary Tumor.</figcaption><p></p>
</figure>
</div>
<p>Here you can see the results per patient.</p>
<p><iframe src="https://uppsala.app.box.com/embed/s/h3rs5sctm0c559yw8qyer235deh22yri?sortColumn=date&amp;view=list" style="width:100%; height:700px;"></iframe></p>
</div>
</div>
</div>
</section>
<section id="nanostring-data" class="level2">
<h2 class="anchored" data-anchor-id="nanostring-data">Nanostring data</h2>
</section>
<section id="survival-data" class="level2">
<h2 class="anchored" data-anchor-id="survival-data">Survival Data</h2>
<p><iframe src="https://uppsala.app.box.com/embed/s/4vuofjoibbam64ycrbwn20h5agcirpc8?sortColumn=date&amp;view=list" style="width:100%; height:700px;"></iframe></p>
</section>
</section>
<section id="data-responsibility" class="level1">
<h1>Data Responsibility</h1>
<ul>
<li><strong>NBIS &amp; Uppnex</strong> Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.</li>
<li><strong>Sensitive data</strong> Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.</li>
<li><strong>Long-term backup</strong> The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.</li>
</ul>
</section>
<section id="acknowledgements" class="level1">
<h1>Acknowledgements</h1>
<p>If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.</p>
<ul>
<li><p><strong>NBIS Staff</strong> are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g.&nbsp;<a href="http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html">ICMJE recommendations</a>. If applicable, please include <strong>Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University</strong>, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: <a href="https://www.nbis.se/resources/support.html">“Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged”</a>.</p></li>
<li><p><strong>UPPMAX</strong> If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: <a href="https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/">“The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ”</a>.</p></li>
<li><p><strong>NGI</strong> In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: <a href="https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication">“The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure”</a>.</p></li>
</ul>
</section>
<section id="closing-procedures" class="level1">
<h1>Closing Procedures</h1>
<p>You should soon be contacted by one of our managers, <a href="jessica.lindvall@nbis.se">Jessica Lindvall</a> or <a href="henrik.lantz@nbis.se">Henrik Lantz</a>, with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within <strong>30 days</strong> the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.</p>
<p>You are naturally more than welcome to come back to us with further data analysis request at any time via <a href="http://nbis.se/support/support.html">NBIS support</a>.</p>
<p>Thank you for using NBIS and all the best for future research.</p>

</section>

</main>
<!-- /main column -->
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>