---
title: "Tumor progression in breast cancer reflected in gene expression patterns and mutational profile"
subtitle: "Project #5975"
author: "Vincent van Hoef"
date: today
title-block-banner: true
reference-location: margin
citation-location: margin
bibliography: ./assets/citations.bib
csl: ./assets/mla.cls
format:
    html:
        theme: cosmo
        toc: true
        toc-location: left
        css: "./assets/style.css"
        self-contained: false
        include-before-body: ./assets/custom.html
execute: 
  echo: false
  warning: false
comments: 
    hypothesis: true
---

```{r}
library(DT)
library(boxr)
box_auth()
```

# Introduction

This is the report of the analysis for project #5975, requested by Lisa Ryd√©n (LU). For this study, targeted next generation sequencing by hybridization capture was performed to get data about a panel of approx 450 genes. The main aim of the analysis is to call somatic variants in a collection of 205 breast cancer samples; some of which are matched pairs of primary tumors and distance metastases or primary tumors and lymph node metastases. Matched normal samples are however not available but might be included later for more accurate results. 

Contract:

* Preprocessing of 205 samples 
* CNV and SNV detection
* Format data to allow for analysis of correlation with survival and clinical annotation
* Detection of differences in CNV and SNV between primary tumors and lymph node or distant metastases
* Correlation with tumor grouping and BC360 gene signatures based on Nanostring expression data

[150 hours]

:::{.callout-note}
You can leave comments on this report by opening the sidebar on the very right of this page. After quickly signing up for Hypothes.is via the link at the top of the sidebar, commenting is quick and easy. Just highlight the section you want to remark upon and write your comment. Leave it in the public group and I will be able to see it and update the text if necessary.
:::

205 samples are included in this study. [The sample table contains 207 entries, but 2 samples were not sequenced.]{.aside} sample 11, 12 and 19 are corrupted and were removed because they cause the analysis pipeline to stop in error. Requested acces to the original files through Celemics.

```{r}
meta <- box_read_csv("955662091441")
datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
```


# Material and Methods

## Preprocessing

Raw reads were trimmed using TrimGalore v0.6.1 prior to mapping to GRCh37 using bwa mem of bwa v0.7.17. Subsequently, duplicates were tagged and removed using MarkDuplicates from picard v2.23.4. Base recalibration was performed and applied using GATK v4.2.0 with known sites datasets dbsnp_138, 1000G_phase1 and Mills_and_1000G_gold_Standard_indels.

## Variant Calling

Somatic variant calling of all samples was performed by combining the results of three variant callers; i.e. Mutect2 v4.2.0, VarScan v2.4.2 and freebayes v1.3.2. These were chosen because they are capable of running without a matched normal sample. Mutect2 was supplied with a generic panel-of-normals ("somatic-b37-Mutect2-exome-panel") and a germline resource ("gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only"). VarScan was run with standard parameters and freebayes was run with -F 0.01 -C 5 --pooled-continuous and subsequently filtered to retain calls with QUAL > 20. Variant calling was restricted to the list of targets used for the hybrid capture. Variants found in all three callers were called again using the *call* command from bcftools 1.14 and annotated with VEP v99 using the *vcf2maf.pl* script from vcf2maf v1.6.21.

Additionally, somatic calling for the distant and lymph node metastases was done in a more rigorous manner. For these samples, the matched primary tumor was used as a *pseudo* normal sample and variant calling was performed by Mutect2 only. A custom panel-of-normals was created from the primary tumors using the *CreateSomaticPanelOfNormals* command from GATK. The same germline resource was used as the one employed for the primary tumor analysis. Contamination was estimated and Mutect2 calls were filtered as per the appropriate [GATK workflow](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132). Keep in mind that this protocol only detects variants that are a variant in the metastatic sample but not in the primary tumor. It does not tell us anything about the variants of the primary tumor in itself; if both primary and metasasis have the variant it is not withheld in this analysis.

## Variant Filtering for Plotting

In addition to taking the intersect of the results of the three variant callers, only variants which are present at a variant allele frequency (VAF) < 1% in the Genome Aggregation Database (gnomAD) OR absent from gnomAD OR present in the COSMIC database AND  with a gnomAD VAF < 1% were retained for plotting purposes.

## Copy Number Analysis

For a rough estimation of the copy number alterations in the metastases, the R package pureCN was used [@riester_purecn:_2016]. It was designed for hybrid capture sequencing data, especially with medium-sized targeted gene panels without matching normal samples in mind (matched whole-exome data is of course supported). The primary tumors were used to create a *pseudo* normal coverage profile which is then compared to the metastatic sample. Keep in mind that this means that any copy number alteration is a change in the metastasis as compared to the primary tumor. It does not tell us anything about the status of the primary tumor in itself.

Analysis of the copy number alterations in the primary tumors was considered to be too unreliable without a matched normal sample.

## Tumor Mutational Burden

To calculate the TMB, the R package TMBLeR was used [@fancello_tmbler_2021]. The TMB is defined here as the number of mutations per Mb, excluding known cancer mutations but including synonymous mutations. There are several ways to define and filter the TMB. The filtering used here is described as well in the TMBLeR's manual. The results of this analysis were correlated with the lymph node status.

# Results

Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Therefore, a quality control step was run on the raw reads before and after trimming by the tool [FastqQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/){target="_blank"}. Three samples are corrupted and were removed (11,12 and 19).

## Quality control 

### FastQC

::: {.panel-tabset}

# Before Trimming

FastQC was run on the raw sequences as they are delivered by the sequencing facility. Read1 and Read2 are considered separately. Here we mainly look for outlier samples.

The full report can be downloaded [here](https://uppsala.box.com/s/hqc0smh7zhxs72pfq8syzchdotl7szv4) via the Download button in Box.

# After Trimming

The tool was run again after removal of low quality bases and adapter sequences.

The full report can be downloaded [here](https://uppsala.box.com/s/6m3aayzwcopwsiefi7cmu0nejbcfcdkk) via the Download button in Box.

:::

Overall, the quality is sufficient to proceed with the analysis.

### Coverage

The number of reads overlapping a specific nucleotide is an important quality metric because it will influence the confidence in the subsequent variant calls. Moreover, the more reads cover the target, the better we will be able to detect small clones. After mapping the trimmed reads to the genome, the average read depth per target (7455 targets in total) and per sample is calculated. The results per sample are shown in @fig-coverage-plot.

#::: {#fig-coverage-plot}
#<iframe src="https://uppsala.app.box.com/embed/s/vidv6u2pfzxlr6rilt5w9evot29ywvcq?sortColumn=date&view=list" style="width:100%; height:500px;"></iframe> 
#Coverage per Sample.
#:::

Sample 224 and 28 have rather low coverage (22x and 33x respectively), but are kept in for the further downstream analysis. But good to keep in mind. The other samples have an average coverage beween 250x and 500x which is sufficient for somatic variant calling in heterogeneous tumors.

## Variant Calling

Here you can find the results of the variant calling analysis for the different sample groups.

### Primary Tumor

The primary tumors have been called by three different variant callers and the intersect of the SNPs of these three tools was retained for further analysis. The standard format to store variant information is as VCF files.

<iframe src="https://uppsala.app.box.com/embed/s/qel9xgwg7brlcld3ckzxnynmsc2qhwol?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>


::: {.panel-tabset}

# Overview

![Overview of Primary Tumor Samples](./Results/variant_calls_all/merged_calls/Plots/3_common/Distant_Tumor_Summary_3_common.svg){#fig-pt-overview width="100%" height="700px"}

# Oncoplot

::: {#fig-pt-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/lsxwmjnlb39a8r2lb63qb5nk1qnkjykh?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Oncoplot of Primary Tumor samples.
:::

# Interaction

::: {#fig-pt-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/qlip1l5vhzabd0myuv43did8hy00l6w1?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Co-occurrence of common mutations.
:::

:::

### Distant Metastases

<iframe src="https://uppsala.app.box.com/embed/s/0t143y75tyaubwajjbbvjwj6f0r606y5?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>

::: {.panel-tabset}

# Overview

::: {#fig-dt-overview}
<iframe src="https://uppsala.app.box.com/embed/s/4pmde9i58cupacftv4aq35l966a4y7wz?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Overview of Distant Tumor samples.
:::

# Oncoplot

::: {#fig-dt-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/62pfkzlt6d6pvd2cyjz7xdcjmn5asflu?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Oncoplot of Distant Tumor samples.
:::

# Interaction

::: {#fig-dt-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/5jb5kn70sk4gnhbleoqmla5psn0hrbrl?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Co-occurrence of common mutations in Distant Tumors.
:::

:::

### Lymph Node Metastases

<iframe src="https://uppsala.app.box.com/embed/s/mst7xyg5uyi98zm2k9216nzaeuz1umtm?sortColumn=date&view=list" style="width:100%; height:400px;"></iframe>

::: {.panel-tabset}

# Overview

::: {#fig-lnm-overview}
<iframe src="https://uppsala.app.box.com/embed/s/fpw76dl8889f6r7q3zqdghbs592h0mne?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Overview of Lymph Node samples.
:::

# Oncoplot

::: {#fig-lnm-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/0wbvhg32r4gp27v4dfuu2wgk2wm1d27m?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Oncoplot of Lymph Node samples.
:::

# Interaction

::: {#fig-lnm-oncoplot}
<iframe src="https://uppsala.app.box.com/embed/s/pqhul5us0o4vg4vq2pqle1rl5f1l79ey?sortColumn=date&view=list" style="width:100%; height:700px;"></iframe>
Co-occurrence of common mutations in Lymph Node samples.
:::

:::

## Tumor Mutational Burden

```{r}
tmb <- box_read_csv("955667979176")
tmb <- tmb[tmb$Filter == "vaf=0NoCancerMuts", !colnames(tmb) %in% "Design"]
datatable(tmb, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE, scrollX = TRUE))
```

## Copy Number Alteration

Copy number alterations were estimated using the R package pureCN. In this analysis, metastases were compared to their matched primary tumor. It is important to keep in mind that therefore these results can be affected by the state of the primary tumor as well; e.g. an amplification in a lymph node tumor might indicate a deletion in the matched primary tumor rather than a genuine amplification.

### Distant Metastases

#```{r}
#dmag <- read.csv("./Results/CNA/distant_tumor/all_genes_alt.csv", sep = ";")
#datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
#```

The full results of the pureCN analysis per patient can be downloaded with the button below. There are several files per patient here and not all are even informative. The per gene statistics for each sample can for example be found in the *_genes.csv files. A description of the columns in this genes.csv file can be found in Table 2 of [the pureCN manual](https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf).

<iframe src="https://uppsala.app.box.com/embed/s/vfavpoydsc4814f7l0k5yemu1urvdx4c?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>

::: {#fig-cna-dm height="800px"}

::: {.panel-tabset}
# Amplified
<iframe src="https://uppsala.app.box.com/embed/s/dc43uc8id7czovj6t9p1x5ij83mo25en?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>
# Deleted
<iframe src="https://uppsala.app.box.com/embed/s/o8zrd9gq2l9h2p7nzvzev0iykaa92083?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>
:::
Most Common Alterations in Distant Tumor versus Primary Tumor.
:::

### Lymph Node Metastases

#```{r}
#dmag <- read.csv("./Results/CNA/distant_tumor/all_genes_alt.csv", sep = ";")
#datatable(meta, rownames = FALSE, extensions = c('Buttons', 'Scroller'), options = list(dom = 'Bfrtip', buttons = c("copy", "csv", "excel"), deferRender = TRUE, scrollY = 200, scroller = TRUE))
#```

The full results of the pureCN analysis per patient can be downloaded here. There are several files per patient here and not all are even informative. The per gene statistics for each sample can for example be found in the *_genes.csv files. A description of the columns in this genes.csv file can be found in Table 2 of [the pureCN manual](https://bioconductor.org/packages/release/bioc/vignettes/PureCN/inst/doc/PureCN.pdf).

<iframe src="https://uppsala.app.box.com/embed/s/kkzcsx0nv9hdgz677mwe5jsd657h3fhw?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>

::: {#fig-cna-lnm height="800px"}

::: {.panel-tabset}
# Amplified
<iframe src="https://uppsala.app.box.com/embed/s/n6340rxrl1lpmyifdn0he3uo9nxa1buy?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>
# Deleted
<iframe src="https://uppsala.app.box.com/embed/s/o70p89hzu6p5dsvi1yrjsuln16408wne?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>
:::
Most Common Alterations in Lymph Node Sample versus Primary Tumor.
:::

## ER Mutations

## Nanostring data

## Survival Data

<iframe src="https://uppsala.app.box.com/embed/s/avpbe9jm9u18tgy88yev1to54ak2bv2u?sortColumn=date&view=list" style="width:100%; height:600px;"></iframe>

# Data Responsibility

- **NBIS & Uppnex** Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.
- **Sensitive data** Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.
- **Long-term backup** The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

# Acknowledgements

If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.

- **NBIS Staff** are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. [ICMJE recommendations](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html). If applicable, please include **Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University**, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: ["Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged"](https://www.nbis.se/resources/support.html).

- **UPPMAX** If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: ["The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ"](https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/).

- **NGI** In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: ["The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure"](https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication).

# Closing Procedures

You should soon be contacted by one of our managers, [Jessica Lindvall](jessica.lindvall@nbis.se) or [Henrik Lantz](henrik.lantz@nbis.se), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within **30 days** the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via [NBIS support](http://nbis.se/support/support.html).

Thank you for using NBIS and all the best for future research.