---
title: "Tumor progression in breast cancer reflected in gene expression patterns and mutational profile"
subtitle: "Project #5975"
author: "Vincent van Hoef"
date: today
title-block-banner: true
reference-location: margin
citation-location: margin
bibliography: ./assets/citations.bib
csl: ./assets/mla.cls
format:
    html:
        theme: cosmo
        toc: true
        toc-location: left
        css: "./assets/style.css"
---

# Introduction

This is the report of the analysis for project #5975, requested by Lisa RydÃ©n (LU).

Contract:

* Preprocessing of 205 samples 
* CNV and SNV detection
* Format data to allow for analysis of correlation with survival and clinical annotation
* Detection of differences in CNV and SNV between primary tumors and lymph node or distant metastases
* Correlation with tumor grouping and BC360 gene signatures based on Nanostring expression data

[150 hours]

# Material and Methods

## Preprocessing

Raw reads were trimmed using TrimGalore v0.6.1 prior to mapping to GRCh37 using bwa mem of bwa v0.7.17. Subsequently, duplicates were tagged and removed using MarkDuplicates from picard v2.23.4. Base recalibration was performed and applied using GATK v4.2.0 with known sites dbsnp_138, 1000G_phase1 and Mills_and_1000G_gold_Standard_indels.

## Variant Calling

Somatic variant calling of primary tumors was performed by combining the results of three variant callers; i.e. Mutect2 v4.2.0, VarScan v2.4.2 and freebayes v1.3.2. These were chosen because they are capable of running without a matched normal semple. Mutect2 was supplied with a panel of normals ("somatic-b37-Mutect2-exome-panel") and a germline resource ("gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only"). VarScan was run with standard parameters and freebayes was run with -F 0.01 -C 2 --pooled-continuous and subsequently filtered to retain calls with QUAL > 20. Variant calling was restricted to the list of targets used for the hybrid capture. Variants found in all three callers were called again using the *call* command from bcftools 1.14 and annotated with VEP v99 using the *vcf2maf.pl* script from vcf2maf v1.6.21.

Somatic calling for the distant and lymph node metastases was done slightly different. For these samples, the matched primary tumor was used as a *pseudo* normal sample and variant calling was performed by Mutect2. A custom panel-of-normals was created from the primary tumors using the *CreateSomaticPanelOfNormals* command from GATK. The same germline resource was used as the one employed for the primary tumor analysis. Contamination was estimated and Mutect2 calls were filtered as per the appropriate [GATK workflow](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132).

## Filtering

In addition to taking the intersect of the results of the three variant callers, variants which are present at a variant allele frequency (VAF) > 1% in the Genome Aggregation Database (gnomAD) were removed. Files with raw nd filtered variant calls are provided witht his report. 

## Copy Number Analysis

For a rough estimation of the copy number alterations in the metastases, the R package pureCN was used [@riester_purecn:_2016]. The primary tumors were used to create a *pseudo* normal coverage profile.

## Tumor Mutational Burden

To calculate the TMB, the R package TMBLeR was used [@fancello_tmbler_2021]. The TMB is defined here as the number of mutations per Mb, excluding known cancer mutations but including synonymous mutations. There are several ways to define the TMB. The one used here is described as well in the TMBLeR's manual.

# Results

Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Therefore, a quality control step was run on the raw reads before and after trimming by the tool [FastqQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/){target="_blank"}.

## Quality control 

::: {.panel-tabset}

# Before Trimming

FastQC was run on the raw sequences as they are delivered by the sequencing facility. Read1 and Read2 are considered separately.

::: {#fig-pre-trim-qc}

<iframe src="./Results/QC/multiqc_report.html"></iframe>

Pre Trim QC
:::

# After Trimming

The tool was run again after removal of low quality bases and adapter sequences.

::: {#fig-pre-trim-qc}

<iframe src="./Results/QC/multiqc_report_after_trim.html"></iframe>

Post Trim QC
:::

:::

## Variant Calling

::: {.panel-tabset}

# Primary Tumor

::: {.panel-tabset}

# Overview

::: {#fig-pt-overview}
<iframe src="./Results/all_samples/merged/3_common/Primary_Tumor_Summary_3_common.pdf"></iframe>
Overview of Primary Tumor samples
:::

# Oncoplot

::: {#fig-pt-oncoplot}
<iframe src="./Results/all_samples/merged/3_common/oncoplots/Distant_Tumor_Oncoplot_3_common.pdf"></iframe>
Oncoplot of Primary Tumor samples
:::

::::

# Distant Metastases

# Lymph Node Metastases

:::

## Tumor Mutational Burden

## Copy Number Alteration

# Data Responsibility

- **NBIS & Uppnex** Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.
- **Sensitive data** Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.
- **Long-term backup** The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

# Acknowledgements

If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.

- **NBIS Staff** are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. [ICMJE recommendations](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html). If applicable, please include **Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University**, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: ["Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged"](https://www.nbis.se/resources/support.html).

- **UPPMAX** If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: ["The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ"](https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/).

- **NGI** In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: ["The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure"](https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication).

# Closing Procedures

You should soon be contacted by one of our managers, [Jessica Lindvall](jessica.lindvall@nbis.se) or [Henrik Lantz](henrik.lantz@nbis.se), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within **30 days** the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via [NBIS support](http://nbis.se/support/support.html).

Thank you for using NBIS and all the best for future research.